### PR TITLE
Fix clang-tidy on CI

### PR DESCRIPTION
**Pull request by nilsvu**
### Code review checklist

- [ ] The code is documented and the documentation renders correctly. Run
  `make doc` to generate the documentation locally into `BUILD_DIR/docs/html`.
  Then open `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).
- [ ] The PR lists upgrade instructions and is labeled `bugfix` or
  `new feature` if appropriate.

### CODE REVIEW

**Review by kidder — DISMISSED** [review]

**Comment by knelli2** [comment]
Seems like this fixed clang-tidy!

**Comment by kidder** [comment]
no, clang-tidy didn't run on anything.  I took the changes in this PR and applied them to one of my PRs that does trigger a clang-tidy run and it didn't fix the problem

**Comment by knelli2** [comment]
Oh RIP... 😢

File: tools/SpectrePch.hpp [hunk 1]
```diff
@@ -27,4 +26,7 @@
 // Include Brigand related headers
 #include <Utilities/TMPL.hpp>
 
+// For Blaze error handling (see SetupBlaze.cmake)
+#include <Utilities/BlazeExceptions.hpp>
```
Comment (nilsdeppe) [inline tools/SpectrePch.hpp]: I think we need to include this before we include blaze. I'm on mobile so maybe I just can't see the file quite right

**Review by nilsdeppe — CHANGES_REQUESTED** [review]

File: src/Utilities/BlazeExceptions.hpp [hunk 2]
```diff
@@ -1,6 +1,8 @@
 // Distributed under the MIT License.
```
Comment (nilsdeppe) [inline src/Utilities/BlazeExceptions.hpp:1]: Code looks good! Sorry I missed this last time, but could you add the hpp to the CMake lists?

Comment (nilsvu) [reply src/Utilities/BlazeExceptions.hpp:1]: Fixed 👍

**Review by nilsdeppe — APPROVED** [review]

**Pull request merged**